### PR TITLE
Added Jitsi audio coniguration buttons

### PR DESCRIPTION
**Pull request by DashieTM**
image

This is just backend for implementation in element-web.
Implemented in: https://github.com/vector-im/element-web/pull/21712
Feature request: https://github.com/vector-im/element-web/issues/21687

### **Notes:**
- Added Buttons as well as EN and DE localization for jitsi audio configuration: echo cancellation, noise suppression, automatic gain, high pass filter
- DE localization for video rooms

Type:[enhancement]

Signed-off-by: DashieTM <fabio.lenherr@gmail.com>

---
Here's what your changelog entry will look like:

## ✨ Features
 * Added Jitsi audio coniguration buttons ([\#8255](https://github.com/matrix-org/matrix-react-sdk/pull/8255)). Contributed by @DashieTM.

Preview: https://pr8255--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

File: src/settings/Settings.tsx [hunk 1]
```diff
@@ -270,6 +270,41 @@ export const SETTINGS: {[setting: string]: ISetting} = {
         // Reload to ensure that the left panel etc. get remounted
         controller: new ReloadOnChangeController(),
     },
+    "feature_disableAEC": {
```
Comment (turt2live) [inline src/settings/Settings.tsx:273]: These probably shouldn't be features but rather settings

File: src/i18n/strings/de_DE.json [hunk 2]
```diff
@@ -3617,6 +3617,12 @@
     "How can I leave the beta?": "Wie kann ich die Beta verlassen?",
     "To feedback, join the beta, start a search and click on feedback.": "Um Feedback zu geben, tritt der Beta bei, starte eine Suche und klick auf Feedback.",
     "Thank you for trying the beta, please go into as much detail as you can so we can improve it.": "Danke dir für das Ausprobieren der Beta, bitte gehe so sehr ins Detail wie du kannst, damit wir es verbessern können.",
+    "Video rooms (under active development)": "Video-Räume (unter Konstruktion)",
+    "Disable Acoustic Echo Cancellation (AEC)": "Automatische Echo Unterdrückung aus(für Video-Räume)",
+    "Disable Noise Supression": "Geräuschunterdrückung aus(für Video-Räume)",
+    "Disable Audio Processing": "Audioverarbeitung aus(für Video-Räume)",
+    "Disable Automatic Gain Control": "Automatische Pegelverstärkung aus(für Video-Räume)",
+    "Disable High Pass Filter": "Hochpassfilter aus(für Video-Räume)",
```
Comment (turt2live) [inline src/i18n/strings/de_DE.json:3625]: Changes to the translation files themselves cannot be done over PR.

**Review by turt2live — COMMENTED** [review]
Thanks for taking a look at this! Unfortunately it requires product/design to take a look before we can merge it, as it's a feature set we might want to change a bit.

I've included general review down below, though more in-depth code review will take place once the product/design team take a look.

File: src/settings/Settings.tsx (hunk 1 above)
Comment (DashieTM) [reply src/settings/Settings.tsx:273]: I had a small debate with myself over where to put them, they are settings but also closely tied to the video rooms, that's why I put them in there. But I can resettle them if that is better.

**Pull request closed without merging**